ford/sourceform.py

```diff
@@ -2532,3 +2537,50 @@ def get_name(self,item):
             return name
 
 namelist = NameSelector()
+
+
+class ExtModule(FortranModule):
```

Comment:
Rename suggestion for all these classes: `Ext*` -> `External*`

Reply:
Yes, fine with me.